package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "flower-card",
+  "version": "1.0.0",
+  "description": "Custom flower card for https://github.com/Olen/homeassistant-plant",
+  "keywords": [
+    "home-assistant",
+    "homeassistant",
+    "lovelace",
+    "custom-cards"
+  ],
+  "module": "room-card.js",
```

Comment:
Is this a typo?

Reply:
Yes haha. Copy pasted it

Reply:
How do I change the code ?

type: custom:flower-card
entity: plant.plant_living_room
show_bars:
  - illuminance
  - moisture
  - conductivity
  - temperature
battery_sensor: sensor.plant_living_room_battery